vocabularies/borehole-sub-purpose.ttl

```diff
@@ -403,7 +402,22 @@ bhfunc:service a skos:Concept ;
     skos:topConceptOf <http://linked.data.gov.au/def/borehole-sub-purpose> .
 
 bhfunc:gas-management a skos:Concept ;
-    skos:definition "Borehole drilled for gas analysis. Typical applications include exploration, compliance and greenhouse gas emissions."@en ;
+    skos:definition "Borehole drilled for gas analysis and management. Typical applications include exploration, management of gas in coal mines, compliance and greenhouse gas emissions."@en ;
     skos:inScheme <http://linked.data.gov.au/def/borehole-sub-purpose> ;
     skos:prefLabel "Gas Management"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/borehole-sub-purpose> .
+
+bhfunc:pggd-functions a skos:Collection ;
+    skos:definition "The functions of boreholes specific to petroleum, greenhouse gas, and geothermal wells and bores i.e. those that require a statutory notification in Queensland regulation."@en ;
+    skos:member bhfunc:water-supply,
+        bhfunc:water-injection,
+        bhfunc:stratigraphic,
+        bhfunc:stimulation,
+        bhfunc:petroleum-reservoir-characterisation,
+        bhfunc:petroleum-production,
+        bhfunc:petroleum-prod-test,
+        bhfunc:petroleum-injection,
+        bhfunc:observation,
+        bhfunc:disposal,
+        bhfunc:heat-flow ;
+    skos:prefLabel "Petroleum Borehole Functions"@en .

```

Comment:
Should the preflabel be "Petroleum Borehole Purposes" rather than "Functions"?

Reply:
The sub-purpose moniker was begrudgingly inherited from legacy systems. As it is distinct from purpose rather than **sub**ordinate to.
"Petroleum Borehole Purposes" may also confuse distinction with the Borehole Purpose vocabulary.

Reply:
I think I meant to write "sub-purpose"...

Reply:
```suggestion
    skos:prefLabel "Petroleum Borehole Sub-Purpose"@en .

```